src/cpp/common/channel_filter.h

```diff
@@ -386,7 +388,7 @@ void RegisterChannelFilter(
        FilterType::SetPollsetOrPollsetSet, FilterType::DestroyCallElement,
        FilterType::channel_data_size, FilterType::InitChannelElement,
        FilterType::DestroyChannelElement, FilterType::GetPeer,
-       FilterType::GetChannelInfo, name}};
+       FilterType::GetChannelInfo, name, INT_MAX}};
```

Comment:
Instead of hard-coding this, let's add a new parameter to `RegisterChannelFilter()`.  Otherwise, there's no way for someone internally to define a C++ filter that overrides one of the OSS filters.